apollo-configservice/src/main/java/com/ctrip/framework/apollo/configservice/service/config/ConfigServiceWithCache.java

```diff
@@ -149,6 +161,39 @@ public void handleMessage(ReleaseMessage message, String channel) {
     }
   }
 
+  @Override
+  public ImmutableMap<String, Release> findReleasesByReleaseKeys(Set<String> releaseKeys) {
+    try {
+      ImmutableMap<String, Long> releaseKeyMap = releaseKeyCache.getAll(releaseKeys);
+      if (CollectionUtils.isEmpty(releaseKeyMap)) {
+        return null;
```

Comment:
For collection return values, prefer returning an empty collection like Collections.emptyMap() instead of null.